### ISSUE TITLE
Cursor suggests functionality even when functionality is not present

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15342]
Cursor suggests functionality even when functionality is not present

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Edit Media

#### Issue description

Users who mouse over the "Save" heading on the Edit Media page see a
"move" cursor, which suggests to users that the "Save" section can
be dragged and dropped to somewhere else. However, this is only the case
if additional boxes are shown, re-ordering is not possible if "Save"
is the only visible box (although the element can be dragged, there is
nowhere else to drop it).

The use of consistent and relevant cursors may be important for users
who have a cognitive disability, since cursors give a visual clue as to
an element's functionality. Using the move cursor for elements which
cannot be moved may be confusing or counter-intuitive for users.

##### Issue Code

```
    .js .postbox .hndle, .js .widget .widget-top {


        cursor: move;


    }
```

#### Remediation Guidance

Only allow the `move` cursor to appear to users when re-ordering is
functionally possible.

Ideally, the drag and drop functionality should not be available at all
when only one box is present (i.e. it should not even be possible to
drag that one box), however this may be unrealistic to fix, and
therefore the solution of removing the cursor style will go a long way
towards matching user expectations.

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-78 in Tenon's report

**Comment by afercia** [comment]
If I'm reading this correctly, this applies to the Media Editor which isn't part of Gutenberg. Suggest moving this issue to Trac.

Screenshot:

![Screenshot 2019-05-04 at 17 50 43](https://user-images.githubusercontent.com/1682452/57181541-74c8a600-6e95-11e9-9129-95cc46fe0721.jpg)

**Comment by StommePoes** [comment]
The whole media pages was done as part of the audit I believe because via Gutenberg there was the "Featured Image" button... but everything after clicking that looked like old stuff. There's a bunch of bugs under two components that I think really aren't Gutenberg: Media Dialog and Edit Media.

**Comment by afercia** [comment]
@StommePoes thanks. Various issues from the WPCampus report need to be moved to Trac as they're not strictly related to Gutenberg. We're in the process of reviewing all of them and adding the labels `Media` or `[Type] WP Core But`. Any help in triaging is welcome :)

**Comment by anevins12** [comment]
Moved to Trac: https://core.trac.wordpress.org/ticket/47131 (with screenshot).

**Comment by afercia** [comment]
Closing as moved to Trac.
